config/routes.rb

```diff
@@ -27,6 +27,7 @@
 
   namespace :merchant do
     get "/", to: "dashboard#index"
+    get "/items", to: "items#index"
```

Comment:
I think moving forward we should use resources. Simple fix so don't worry now

Reply:
Yeah, I agree we should change that eventually